custom_formatter.rb

```diff
@@ -1,11 +0,0 @@
-require "rspec/core/formatters/base_text_formatter"
```

Comment:
This looked like a PoC/example and isn't part of the rspec-teamcity functionality. It looks like a modified example from https://www.relishapp.com/rspec/rspec-core/v/3-3/docs/formatters/custom-formatters
What functionality does this file provide?